server/apps/immich/src/api-v1/asset/asset-repository.ts

```diff
@@ -58,13 +60,17 @@ export class AssetRepository implements IAssetRepository {
     @Inject(TAG_REPOSITORY) private _tagRepository: ITagRepository,
   ) {}
 
-  async getAssetWithNoSmartInfo(): Promise<AssetEntity[]> {
+  async getAssetsWithNoAutoGeneratedTags(): Promise<AssetEntity[]> {
     return await this.assetRepository
       .createQueryBuilder('asset')
-      .leftJoinAndSelect('asset.smartInfo', 'si')
-      .where('asset.resizePath IS NOT NULL')
-      .andWhere('si.id IS NULL')
+      .leftJoinAndSelect('asset.tags', 'tag')
+      .where(
+        new Brackets((qb) => {
+          qb.where('tag.id IS NULL').orWhere(`tag.type = 'CUSTOM'`);
+        }),
+      )
```

Comment:
What's the result of this query when an asset has no tags at all? Will it still be picked up for processing?

Reply:
Yes it will be picked up